### DISCUSSION TITLE
First image flash with pxe server

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1695] [General]
Hello everyone, 

I am using Jetson Orin Nano devkit and using Yocto-OpemEmbeddedCore tools for image build. Normally I can flash the image with force recovery. When the board is in force recovery mode, it act as USB device that's why need to connect via USB. Thats works fine. 

Consider again a empty SD/SSD, is that possible to flash image using PXE server ? I am not talking about reflashing.

**Comment by dwalkes** [comment]
Based on https://docs.nvidia.com/jetson/archives/r36.3/DeveloperGuide/SD/FlashingSupport.html#configuring-a-pxe-boot-server-for-uefi-bootloader-on-jetson and https://forums.developer.nvidia.com/t/pxe-installation-of-jetsons/279247/3?u=danwalkes it's possible although I haven't tried it myself.  Note the comments about needing to flash the bootloader through USB.

**Comment by noob502** [comment]
That means I have to Flash for the first time via USB. I can see the UEFI option in BIOS, but it can not detect any network driver like Windows.
